packages/core/src/builders.test.ts

```diff
@@ -137,6 +137,59 @@ describe("makeReactionRemove", () => {
   });
 });
 
+describe("makeTagAddData", () => {
```

Comment:
nice, guessing you're trying to bring up new tests. Are they passing?

Reply:
yes, they all are